packages/libs/ui/nft-react-hooks/src/lib/utils/useCustomQuery.tsx

```diff
@@ -0,0 +1,30 @@
+import {
```

Comment:
I think this is overkill, have you seen https://www.apollographql.com/docs/react/api/link/apollo-link-error/#options?

You can do this as an afterware step in the client itself

Reply:
Fair enough, included on baf1595!

Reply:
Sweet! Last thing... could you update the docs (they're duplicated in the root and the package) to reflect no API key is needed? Then i think it's good to merge